tests/nodes/primary_nodes/test_project.py

```diff
@@ -95,3 +98,46 @@ def test_integration_project(cript_api, simple_project_node):
 
     # ========= test delete =========
     delete_integration_node_helper(cript_api=cript_api, node_to_delete=simple_project_node)
+
+
+@pytest.mark.skip(reason="api")
+def test_save_project_node(cript_api, simple_project_node, complex_project_node):
+    """
+    pytest nodes/primary_nodes/test_project.py::test_save_project_node
+    """
+
+    # with cript.API(host="https://lb-stage.mycriptapp.org/") as api:
+    #     with open("new_project.json") as json_handle:
+    #         proj_json = json.load(json_handle)
+
+    # proj = cript_api.load_nodes_from_json(nodes_json=proj_json)
+    # Modify deep in the tree
+    print("------------\nstarting")
+    proj0 = copy.deepcopy(complex_project_node)
+    proj_json = proj0.get_json().json
+    cript_api.save_new(proj0)
+
+    proj = cript.load_nodes_from_json(nodes_json=json.dumps(proj_json))
+    print("\n----proj_loaded")
+    print(proj)
+
+    print("------------\n1111111")
+    print(type(proj))
```

Comment:
@InnocentBug Ludwig, if you have a moment do you mind telling me why this might load in a string?

also my work flow and logic in this test , is to ensure that I can save the node as just a "creation" step , and then be able to reload the node back in ( via using "load nodes from json") , 

then the idea to edit it , save it , then load it again ensuring the changes are indeed saved,
but the return value should be a node object right ? this is also what the function docs say and what I remember in my past work. I made this branch off of the current development and that function has not been touched by the changes in this PR so I'm a bit perplexed though its probably something simple , so I'm wondering what your thoughts are

Reply:
It does load a string if it thinks that it isn't a node.
(Or valid JSON)